src/qibo/quantum_info/metrics.py

```diff
@@ -1086,8 +1088,14 @@ def entangling_capability(circuit, samples: int, backend=None):
     return capability
 
 
-def expressibility(circuit, t: int, samples: int, backend=None):
-    """Returns the expressibility :math:`\\|A\\|_{HS}` of a parametrized
+def expressibility(
+    circuit,
+    power_t: int,
+    samples: int,
```

Comment:
It would be good to pass `exact` as input, just like in `haar_integral()`, or use `samples` to decide whether to calculate by sampling or exactly, as @MatteoRobbiati  suggests.

Reply:
Implemented @MatteoRobbiati's suggestion